src/main/java/net/sf/jabref/model/database/BibDatabase.java

```diff
@@ -490,6 +486,28 @@ private String resolveContent(String result, Set<String> usedIds) {
 
 
     /**
+     * Resolves the refLabel with the given information using {@link reSolveString(String label, Set<String> usedIds)}. If the ref label could be resolved, the
+     * # characters will be stripped and then the resolved string will be added to newRes. Otherwise the refLabel string will be added to newRes, which means
+     * that the # characters will be displayed.
+     *
+     * @param res String to resolve
+     * @param newRes StringBuilder to append the labels
+     * @param next ref label position as int
+     * @param stringEnd end position of the label as int
+     * @param usedIds Set of Strings, containing the IDs that are used
+     */
+    private void resolveRefLabel(String res, StringBuilder newRes, int next, int stringEnd, Set<String> usedIds) {
```

Comment:
Reintegrate the method. 5 LOC for a method with 5 paramters. -- no extra method necessary.